script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -582,7 +582,7 @@ This billowing wave of energy so potent that death itself cannot trespass upon i
 [sha:38c66ac6a9f8e6baddbbd1ffebc753651be40418]{
 -- Page 339, Offset 32606.
 -- 身をもって千年の代償を受けるがいい。
-You can experience firsthand the retribution for this past millennia.
+You can experience firsthand the retribution for these past millennia.
```

Comment:
I think it's a singular past 1k years right? Since Roa's been chasing her for like 800?